### DISCUSSION TITLE
请问不使用存储桶可以吗？

### DISCUSSION THREAD

**Discussion by Febtw** [OXeu/Rin#270] [General]
因为没有信用卡，也不需要R2，所以请问搭建时不配置R2可以吗？

**Comment by OXeu** [comment]
可以的，用其他兼容 S3 协议的对象存储桶就行

**Reply by Febtw** [reply]
我的意思是不使用存储桶可以吗？

**Reply by OXeu** [reply]
> 我的意思是不使用存储桶可以吗？

完全不使用存储桶的话不支持通过设置面板动态配置，同时会影响到缓存，其他功能理论上能够正常工作，但是没有经过测试，可能存在一些问题

**Reply by Febtw** [reply]
> 不支持通过设置面板动态配置

那么是在仓库中修改配置吗？

**Reply by OXeu** [reply]
> > 不支持通过设置面板动态配置
> 
> 那么是在仓库中修改配置吗？

如果需要改配置的话确实就需要改代码了
